From e354cee4d1161ce2a51938476aeba2c1d19ac194 Mon Sep 17 00:00:00 2001
From: Maarten Wijnants <wijnants.maarten@gmail.com>
Date: Tue, 1 Apr 2025 10:25:15 +0200
Subject: [PATCH 1/2] Update thunderbird label

---
 fragments/labels/thunderbird.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fragments/labels/thunderbird.sh b/fragments/labels/thunderbird.sh
index 0821c5838..d40504993 100644
--- a/fragments/labels/thunderbird.sh
+++ b/fragments/labels/thunderbird.sh
@@ -1,7 +1,7 @@
 thunderbird)
     name="Thunderbird"
     type="dmg"
-	appNewVersion=$(curl -I -s "https://download.mozilla.org/?product=thunderbird-esr-latest&os=osx&lang=en-US" | grep -i location | grep -oE '[0-9]+\.[0-9]+\.[0-9]+' | head -n1 )
-    downloadURL="https://download.mozilla.org/?product=thunderbird-esr-latest&os=osx&lang=en-US"
+	appNewVersion=$(curl -fsL "https://www.thunderbird.net/en-US/thunderbird/releases/atom.xml" | grep -v "esr" | grep -m 5 "<title type=" | sed 's/<.*>\(.*\)<.*>/\1/' | awk '{ print $2 }' | awk '{$1=$1};1' | sort -V | tail -n1 )
+    downloadURL="https://download.mozilla.org/?product=thunderbird-latest&os=osx&lang=en-US"
     expectedTeamID="43AQ936H96"
     ;;

From edb392a4b3e2bc178e3a3682d89da71503a79a0e Mon Sep 17 00:00:00 2001
From: Maarten Wijnants <wijnants.maarten@gmail.com>
Date: Tue, 1 Apr 2025 11:37:55 +0200
Subject: [PATCH 2/2] Update thunderbird label

---
 fragments/labels/thunderbird.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fragments/labels/thunderbird.sh b/fragments/labels/thunderbird.sh
index d40504993..9729acc32 100644
--- a/fragments/labels/thunderbird.sh
+++ b/fragments/labels/thunderbird.sh
@@ -1,7 +1,7 @@
 thunderbird)
     name="Thunderbird"
     type="dmg"
-	appNewVersion=$(curl -fsL "https://www.thunderbird.net/en-US/thunderbird/releases/atom.xml" | grep -v "esr" | grep -m 5 "<title type=" | sed 's/<.*>\(.*\)<.*>/\1/' | awk '{ print $2 }' | awk '{$1=$1};1' | sort -V | tail -n1 )
     downloadURL="https://download.mozilla.org/?product=thunderbird-latest&os=osx&lang=en-US"
+    appNewVersion=$(curl -fsIL "$downloadURL" | grep -i Location | grep -o "/[0-9.]\+/" | grep -o "[0-9.]\+")
     expectedTeamID="43AQ936H96"
     ;;
